src/main/java/uk/gov/hmcts/reform/bsp/common/mapper/GenericMapper.java

```diff
@@ -25,7 +25,7 @@ public static void addMappingsTo(String parentField, ImmutableMap<String, String
 
     public static Optional<String> getValueFromOcrDataFields(String fieldName, List<OcrDataField> ocrDataFields) {
         return ocrDataFields.stream()
-            .filter(f -> f.getName().equals(fieldName))
+            .filter(f -> f.getName().equals(fieldName) && f.getValue() != null)
```

Comment:
this makes sure we don't have elements on the list with null as a value

Reply:
I'd add a further filter instead. Like ".filter(Objects::nonNull)".